cloud/blockstore/libs/daemon/common/bootstrap.cpp

```diff
@@ -508,6 +509,23 @@ void TBootstrapBase::Init()
         .NbdDevicePrefix = Configs->ServerConfig->GetNbdDevicePrefix(),
     };
 
+    NBD::IDeviceFactoryPtr nbdDeviceFactory = NBD::CreateDeviceFactory(
+        Logging,
+        TDuration::Days(1));    // timeout
+
+    if (Configs->ServerConfig->GetNbdNetlink()) {
+#ifdef NETLINK
+        nbdDeviceFactory = NBD::CreateNetlinkDeviceFactory(
+            Logging,
+            TDuration::Days(1), // timeout
+            TDuration::Days(1), // deadConnectionTimeout
+            true,               // reconfigure
+            false);             // disconnect
```

Comment:
This was my initial idea, but there are 2 problems with it

1. We still want devices configured via ioctl interface to disconnect properly. Otherwise we won't be able to use associated /dev/nbdX device until we fix the driver
2. dev->Stop() is triggered by StopEndpoint which is called for each endpoint when nbs restarts. If we want to disconnect device only for explicitly stopped endpoints, we will need to differentiate between these 2 cases somehow